src/modules/visitor-space/components/AutocompleteFieldInput/AutocompleteFieldInput.tsx

```diff
@@ -0,0 +1,67 @@
+import { type SelectOption } from '@meemoo/react-components';
+import clsx from 'clsx';
+import { type FC, useCallback } from 'react';
+import { type ActionMeta, type SingleValue } from 'react-select';
+import AsyncSelect from 'react-select/async';
+
+import { IeObjectsService } from '@ie-objects/services';
+import { tText } from '@shared/helpers/translate';
+
+import styles from './AutocompleteFieldInput.module.scss';
```

Comment:
Isn't this a nice moment to try the Styles.someclass.another class writing method?

Reply:
We don't have an agreement with all developers on it yet
We shouldn't introduce a new writing convention without everyone being on board, otherwise you work against eachother
For now we'll just keep the naming convention uniform across the codebase 🤷